packages/react/src/Datepicker/shared/utils.js

```diff
@@ -0,0 +1,34 @@
+const dateFormatter = ({
```

Comment:
I don't think it's necessary to add a `shared` folder. You can put the `utils.js` file under the `DatePicker` folder.